src/python/pants/backend/python/rules/targets.py

```diff
@@ -36,11 +36,11 @@ def validate_snapshot(self, snapshot: Snapshot) -> None:
 
 
 class PythonLibrarySources(PythonSources):
-    default_globs = ("*.py", "!test_*.py", "!*_test.py", "!conftest.py")
+    default = ("*.py", "!test_*.py", "!*_test.py", "!conftest.py")
```

Comment:
This is a big win, imo. We're going from a custom implementation of defaults to now one unified mechanism that rule authors can anticipate.